bin/localnet/deployer/deployer.go

```diff
@@ -52,17 +56,67 @@ func main() {
 	checkError("backend lookup", err)
 	<-synced
 
+	// we still need to deploy legacy contracts to blockchain to be backwards compatible with current usage
+	deployLegacyContracts(transactor, client)
+
+	deployPaymentsv2Contracts(transactor, client)
+}
+
+func deployPaymentsv2Contracts(transactor *bind.TransactOpts, client *ethclient.Client) {
```

Comment:
We already write Golang jobs as magefile.go tasks. Would be nice to connect this one there

Reply:
Some day after tomorrow ;)